plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -1782,4 +1783,193 @@ describe('PersonState.update()', () => {
             )
         })
     })
+    describe('on persons merges', () => {
+        it('postgres and clickhouse get updated', async () => {
+            const first: Person = await hub.db.createPerson(
+                timestamp,
+                {},
+                {},
+                {},
+                teamId,
+                null,
+                false,
+                uuid.toString(),
+                ['first']
+            )
+            const second: Person = await hub.db.createPerson(
+                timestamp,
+                {},
+                {},
+                {},
+                teamId,
+                null,
+                false,
+                uuid2.toString(),
+                ['second']
+            )
+
+            const state: PersonState = personState({}, first)
+            jest.spyOn(hub.db.kafkaProducer, 'queueMessages')
+            jest.spyOn(state, 'aliasDeprecated').mockImplementation()
+            await state.mergePeople({
```

Comment:
non-blocking

Given that we may want to refactor the `PersonState` code, I would try to make this test only depend on the a stable top level API, thereby reducing the amount we'll need to update here. For instance [this test that checks handling of postgres errors](https://github.com/PostHog/posthog/blob/master/plugin-server/tests/main/ingestion-queues/run-ingestion-pipeline.test.ts#L43).

Reply:
will not do this here as I have another pr on top already